db/seeds/users.rb

```diff
@@ -1,2 +1,188 @@
+User.create(first: 'Pudding', last: 'Sugar' , email: 'PuddingSugar@Sugar.com', user_name: 'PuddingSugar', street: '96121 Main St.' , city: 'Newtown', state: 'CT', zip: 16474, password: 'password', password_confirmation: 'password', admin: false)
+User.create(first: 'Donut', last: 'Sugar' , email: 'DonutSugar@Sugar.com', user_name: 'DonutSugar', street: '33 Main St.' , city: 'Nashua', state: 'NH', zip: 13062, password: 'password', password_confirmation: 'password', admin: false)
+User.create(first: 'Cotton', last: 'Sugar' , email: 'CottonSugar@Sugar.com', user_name: 'CottonSugar', street: '39887 Main St.' , city: 'Nashua', state: 'NH', zip: 13062, password: 'password', password_confirmation: 'password', admin: false)
+User.create(first: 'Candy', last: 'Sugar' , email: 'CandySugar@Sugar.com', user_name: 'CandySugar', street: '98404 Main St.' , city: 'Topeka', state: 'KS', zip: 94625, password: 'password', password_confirmation: 'password', admin: false)
+User.create(first: 'Brownie', last: 'Sweet' , email: 'BrownieSweet@Sweet.com', user_name: 'BrownieSweet', street: '51369 Main St.' , city: 'Denver', state: 'CO', zip: 80202, password: 'password', password_confirmation: 'password', admin: false)
+User.create(first: 'Jelly', last: 'Sweet' , email: 'JellySweet@Sweet.com', user_name: 'JellySweet', street: '43927 Main St.' , city: 'Ashbury', state: 'AR', zip: 27482, password: 'password', password_confirmation: 'password', admin: false)
+User.create(first: 'Beans', last: 'Tiramisu' , email: 'BeansTiramisu@Tiramisu.com', user_name: 'BeansTiramisu', street: '65852 Main St.' , city: 'Miami', state: 'FL', zip: 66666, password: 'password', password_confirmation: 'password', admin: false)
+User.create(first: 'Chupa', last: 'Tiramisu' , email: 'ChupaTiramisu@Tiramisu.com', user_name: 'ChupaTiramisu', street: '63489 Main St.' , city: 'Denver', state: 'CO', zip: 80202, password: 'password', password_confirmation: 'password', admin: false)
+User.create(first: 'Chups', last: 'Tiramisu' , email: 'ChupsTiramisu@Tiramisu.com', user_name: 'ChupsTiramisu', street: '44165 Main St.' , city: 'Springfield', state: 'MO', zip: 57294, password: 'password', password_confirmation: 'password', admin: false)
+User.create(first: 'Carrot', last: 'Tiramisu' , email: 'CarrotTiramisu@Tiramisu.com', user_name: 'CarrotTiramisu', street: '4521 Main St.' , city: 'Paris', state: 'TX', zip: 44444, password: 'password', password_confirmation: 'password', admin: false)
+User.create(first: 'Cake', last: 'Tiramisu' , email: 'CakeTiramisu@Tiramisu.com', user_name: 'CakeTiramisu', street: '48205 Main St.' , city: 'Nashua', state: 'NH', zip: 13062, password: 'password', password_confirmation: 'password', admin: false)
+User.create(first: 'Chupa', last: 'Tootsie' , email: 'ChupaTootsie@Tootsie.com', user_name: 'ChupaTootsie', street: '45831 Main St.' , city: 'Nashua', state: 'NH', zip: 13062, password: 'password', password_confirmation: 'password', admin: false)
```

Comment:
Line is too long. [246/80]<br>Space found before comma.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.